sparkbox/apprentices-page-next
# SC-1732: Move v16 to past, update employment, remove current apprenticeship

## Description

Changes made per [Kaitlyn's doc](https://docs.google.com/document/d/1oDK3-idFlpXxXXWt8kg904eW906Xe01YyqzqP30QYWA/edit):

- Remove "current apprenticeship" section just for now until we restart
- Move v16 to "past apprenticeship" section
- Update employment status for v16 and others

### Spec

See Story: [SC-1732](https://sparkbox.atlassian.net/browse/SC-1732)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Navigate to the Netlify PR Deploy Preview
3. Make sure the changes align with Kaitlyn's requests in [her document](https://docs.google.com/document/d/1oDK3-idFlpXxXXWt8kg904eW906Xe01YyqzqP30QYWA/edit)

## Validation

The following has been completed by the developer:


[SC-1732]: https://sparkbox.atlassian.net/browse/SC-1732?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
